CI/CD template changes for GitLab/GitHub (#2619)
astronomer/docs: astro/ci-cd-templates/drone.md

Use the following CI/CD templates to automate deploying Apache Airflow DAGs from a Git repository to Astro with [Drone CI](https://www.drone.io/).

> The templates for DroneCI use the [image deploy](template-overview.md#template-types) process with a _single branch implementation_, which requires only one Astro Deployment.

Reviewer: ```suggestion
The template for DroneCI is based on the [image deploy template](template-overview.md#template-types) with a _single branch implementation_, which requires only one Astro Deployment.
```
